agent/engine/docker_task_engine_test.go

```diff
@@ -918,27 +917,15 @@ func TestTaskTransitionWhenStopContainerTimesout(t *testing.T) {
 					}()
 				}).Return(dockerapi.DockerContainerMetadata{DockerID: containerID}),
 
-			// StopContainer times out
-			client.EXPECT().StopContainer(gomock.Any(), containerID, gomock.Any()).Return(containerStopTimeoutError),
-			// Since task is not in steady state, progressContainers causes
-			// another invocation of StopContainer. Return a timeout error
-			// for that as well.
-			client.EXPECT().StopContainer(gomock.Any(), containerID, gomock.Any()).Do(
```

Comment:
the removed code still seems like a valid test case. why not add another test instead of replacing this?

Reply:
Actually the removed code is not valid anymore. That code was explicitly testing that there was no "container event" being emitted, but of course now there is one because this PR is allowing the container to transition to `STOPPED`. Is There a particular use-case that you think is not covered?